docs/src/pages/docs/frequently-asked-questions-page.mdx

```diff
@@ -232,6 +232,29 @@ Metadata attribute using the `label_colors` key.
 }
 ```
 
+### Is there a way to add custom color schemes?
+
+It is possible to override SETUP_EXTRA with COLOR_SCHEMES to register new color schemes
+
+```
+SETUP_EXTRA = {"COLOR_SCHEMES": [
+    {
+        "id": 'myCustomColors',
+        "schemeType": 'Categorical', //Categorical or Sequential
+        "description": '',
+        "label": 'My custom Colors',
+        "colors": ['#6BD3B3', '#6BD3B3', '#6BD3B3', '#66CBE2']
+    }]
+}
```

Comment:
`SETUP_EXTRA` feels ambiguous, and will be difficult/disruptive to change once introduced. Perhaps break this up into `EXTRA_COLOR_SCHEMES` or even `EXTRA_CATEGORICAL_COLOR_SCHEMES` and `EXTRA_SEQUENTIAL_COLOR_SCEHEMES` would make sense?

Reply:
makes sense. added changes accordingly